docs/static/releasenotes.asciidoc

```diff
@@ -65,6 +101,16 @@ No user-facing changes in Logstash core and plugins.
 ** `java.nio.file.NoSuchFileException` when finalizing the segment https://github.com/elastic/logstash/pull/15233[#15233]
 ** DLQ file using wrong sort order https://github.com/elastic/logstash/pull/15246[#15246]
 
+==== Notable plugin issues
```

Comment:
I'm uncomfortable with the "Notable plugin issues" section title.  With 200+ plugins, we're talking on a lot of added responsibility by deciding what is considered "notable."

Reply:
It seems like it would be easier for users to scan the full list, pick out the plugins they care about, and then see the changes for those.

Reply:
It is my thought and I don't know such situations handled in the past. The intention here, to let customer know mentioned plugins have an issue and bring them to the issue details and solution through the linked issue under the plugin repo. Feel free to suggest the better strategy suitable for this case. Thanks.

Reply:
After better understanding your intent, I believe we should add this info to the top of 8.10.3 release notes as "Known issues."

Reply:
We should add the Known Issues to top of 8.10.2, 8.10,1, and 8.10.0 release notes.

Reply:
I have added know imap and email plugin issues to top of 8.10+ sections.
Please review them and if it requires more effort, I am happy to merge without known issues, and make it happen them with separate PR.
Thank you @karenzone !